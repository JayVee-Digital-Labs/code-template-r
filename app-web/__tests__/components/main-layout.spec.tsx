import React, { Component } from 'react'
import { render } from '@testing-library/react'
import MainLayout from '../../components/main-layout'

class SampleComponent extends Component {
  render() {
    return <div>Sample Component</div>
  }
}

describe('Main Layout', () => {
  it('should match snapshot with rendered Child Components', () => {
    const { container } = render(
      <MainLayout>
        <SampleComponent />
      </MainLayout>
    )
    expect(container).toMatchSnapshot()
  })
})
