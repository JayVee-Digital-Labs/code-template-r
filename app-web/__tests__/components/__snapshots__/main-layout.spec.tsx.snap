// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Layout should match snapshot with rendered Child Components 1`] = `
<div>
  <main>
    <div>
      Sample Component
    </div>
  </main>
</div>
`;
